.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
+dist: trusty
 before_install:
   - gem install bundler
-  - gem update bundler
 rvm:
-  - 1.9.3
-  - 2.0
-  - 2.1
+  - 2.3.3
   - 2.2
```

Comment:
Other versions suffer from the same problem you explained in https://github.com/Locale/localeapp/pull/213/commits/fb8e7efdf7aeb6209b0c496130a982ebcf05103e. For example, your build run against `2.2.5`, not `2.2.6`. I normally just specify full versions everywhere. That way I know exactly which version the build runs against and I don't get PR's trying to bring consistency by switching back `2.3.3` to `2.3`.